libpod/runtime_ctr.go

```diff
@@ -918,6 +918,56 @@ func (r *Runtime) PruneContainers(filterFuncs []ContainerFilter) (map[string]int
 	return prunedContainers, pruneErrors, nil
 }
 
+// MountStorageContainer mounts the storage container's root filesystem
+func (r *Runtime) MountStorageContainer(id string) (string, error) {
```

Comment:
You note in the test that this doesn't work rootless - should we check for that in all of the API functions, so we can return sane errors?

Reply:
Well it will work rootless if you do a podman unshare. The problem is we can not do this in the test suite easily.
podman unshare podman mount alpine-working-container
Will work fine.

Reply:
Can we check if we're inside the user namespace?

Reply:
This issue has existed for all podman mounts not just external mounts.  

There already is a check in the podman command.
podman mount -l
Error: cannot run command "podman mount" in rootless mode, must execute `podman unshare` first